django/db/models/sql/query.py

```diff
@@ -227,9 +230,7 @@ def __init__(self, model, alias_cols=True):
 
         self._filtered_relations = {}
 
-        self.explain_query = False
-        self.explain_format = None
-        self.explain_options = {}
+        self.explain_info = None  # type: ExplainInfo | None
```

Comment:
I don't know that the comment-style type hint is something we've got anywhere else. I don't know that a non-comment type hint would be appropriate either though.
Thus, I'm just line-highlighting it for someone to check on.

Reply:
Oh I just wrote this comment for myself whilst thinking things through, I didn't mean it to be part of the final PR. Removing.